tiledb/sm/rest/rest_client.cc

```diff
@@ -465,6 +465,35 @@ Status RestClient::get_array_metadata_from_rest(
       array->unsafe_metadata(), serialization_type_, returned_data);
 }
 
+Status RestClient::get_array_maximum_tile_size_from_rest(
+    const URI& uri, uint64_t* maximum_tile_size) {
+  // Init curl and form the URL
+  Curl curlc(logger_);
+  std::string array_ns, array_uri;
+  RETURN_NOT_OK(uri.get_rest_components(&array_ns, &array_uri));
+  const std::string cache_key = array_ns + ":" + array_uri;
+  RETURN_NOT_OK(
+      curlc.init(config_, extra_headers_, &redirect_meta_, &redirect_mtx_));
+  const std::string url = redirect_uri(cache_key) + "/v1/arrays/" + array_ns +
```

Comment:
Are we adding a new route for this?

Reply:
not sure what you mean by 'route', but I assume support must be added to the REST server to handle the request...

Reply:
Created https://app.shortcut.com/tiledb-inc/story/23814/rest-server-support-for-maximum-tile-size-needed.